### PR TITLE
Code refactoring for client package

### LINKED ISSUE #73: Coe refactoring for client package
## Feature Request

**Is your feature request related to a problem? Please describe.**  

**Describe the solution you'd like**  

Split the code into the separated directory followed by the feature of each function of the `client` package.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by anthonyminyungi**
# Summary 

- It closes #73 

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: packages/client/src/type/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export enum StorageType {
+  LocalStorage = 'LOCAL_STORAGE',
+  IndexedDB = 'INDEXED_DB',
+  ChromeExtension = 'CHROME_EXTENSION',
+}
+
+export enum DataType {
+  Key = 'KEY',
+  Message = 'MESSAGE',
+}
```
Comment (anthonyminyungi) [inline packages/client/src/type/index.ts:17]: Moved from `src/utils.ts`

**Review by anthonyminyungi — COMMENTED** [review]
Related to #75, #77

File: packages/client/src/util/decryptMessage.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { AES, enc } from 'crypto-js';
+
+export function decryptMessage(encryptedMessage: string, key: string): string {
+  return AES.decrypt(encryptedMessage, key).toString(enc.Utf8);
+}
```
Comment (HyeockJinKim) [inline packages/client/src/util/decryptMessage.ts:5]: Make `util` folder's name to `utils`.

**Review by HyeockJinKim — CHANGES_REQUESTED** [review]
Take a look review.

File: packages/client/src/utils/encryptMessage.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { AES } from 'crypto-js';
+
+export function encryptMessage(message: string, key: string): string {
```
Comment (anthonyminyungi) [inline packages/client/src/utils/encryptMessage.ts:3]: Directory name was changed `util` -> `utils`.

**Review by anthonyminyungi — COMMENTED** [review]
Review reflected.

**Review by HyeockJinKim — APPROVED** [review]
LGTM

**Pull request merged**